idea-plugin/build.gradle.kts

```diff
@@ -33,7 +33,13 @@ dependencies {
 }
```

Comment:
We should cherry-pick it to `master` after merging.

(usually we merge to `master` first and cherry-pick to the release)